extension/chrome/settings/setup.ts

```diff
@@ -97,14 +97,14 @@ Catch.try(async () => {
       if (!GoogleAuth.hasReadScope(storage.google_token_scopes || [])) {
         $('.auth_denied_warning').css('display', 'block');
       }
-      if (typeof storage.addresses === 'undefined') {
+      if (typeof storage.sendAs === 'undefined') {
         if (GoogleAuth.hasReadScope(storage.google_token_scopes || [])) {
-          Settings.fetchAcctAliasesFromGmail(acctEmail).then(saveAndFillSubmitOption).catch(Catch.reportErr);
+          Settings.fetchAcctAliasesFromGmail(acctEmail, true).then(saveAndFillSubmitOption).catch(Catch.reportErr);
         } else { // cannot read emails, don't fetch alternative addresses
```

Comment:
Which means you can remove this else branch completely.

Reply:
Got it, removed